client/webpack/common.config.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
 			},
 		],
 	},
+	output: {
+		publicPath: "/",
+	},
```

Comment:
why did you change this file ?

Reply:
that's John Sharpe's bufix foe some heroku stuff